pkg/controller/namespace/deletion/namespaced_resources_deleter.go

```diff
@@ -177,7 +179,7 @@ func (d *namespacedResourcesDeleter) initOpCache() {
 			verbs := sets.NewString([]string(r.Verbs)...)
 
 			if !verbs.Has("delete") {
-				klog.V(6).Infof("Skipping resource %v because it cannot be deleted.", gvr)
+				logger.V(6).Info("Skipping resource because it cannot be deleted", "resource", gvr)
```

Comment:
```suggestion
				logger.V(6).Info("Skipping resource because it cannot be deleted.", "resource",  gvr)
```

Reply:
thanks for you suggestion. but I think we don't need dots and extra spaces